vms/components/fees/config.go

```diff
@@ -0,0 +1,15 @@
+// Copyright (C) 2019-2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package fees
```

Comment:
components/fees contains data structures and code shared among P-chain and X-chain.